assets/create-template/templates/default/template/package.json.js

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
I'm not sure about the naming of this file and its purpose

Reply:
Removed thanks @Amzani